ArduPlane/takeoff.cpp

```diff
@@ -29,6 +29,24 @@ bool Plane::auto_takeoff_check(void)
     }
 
     takeoff_state.last_check_ms = now;
+    
+    //check if waiting for rudder neutral after rudder arm
+    if (plane.arming.last_arm_method() == AP_Arming::Method::RUDDER &&
```

Comment:
Any reason not to do this for all arming types? Stick arming "guarantees" a none neutral rudder but you could have it from other methods also.